lib/membership/member.js

```diff
@@ -178,6 +178,10 @@ Member.prototype._isLocalOverride = function _isLocalOverride(update) {
         return false;
     }
 
+    if (this.incarnationNumber > update.incarnationNumber) {
+        return false;
```

Comment:
Can we create a unit test for this branch? Looking at `test/unit/member_test.js`, I think it should be quite easy.